modules/hcp-migrate-aws-single-to-multiarch.adoc

```diff
@@ -0,0 +1,164 @@
+
+A single-architecture hosted cluster can manage node pools of only one particular CPU architecture. However, a multi-architecture hosted cluster can manage node pools with different CPU architectures. On {aws-short}, a multi-architecture hosted cluster can manage both 64-bit AMD and 64-bit ARM node pools.
+
+.Prerequisites
+
+Before transitioning a hosted cluster from single-architecture to multi-architecture, review the following requirements:
+
+* You have installed an {product-title} management cluster for {aws-short} on {rh-rhacm-first} with the {mce}.
+* You have an existing single-architecture hosted cluster that uses 64-bit AMD variant of the {product-title} release payload.
+* An existing node pool that uses the same 64-bit AMD variant of the {product-title} release payload and is managed by an existing hosted cluster.
+* Ensure that you installed the following command-line tools:
+
+** `oc`
+** `kubectl`
+** `hcp`
+** `skopeo`
+
+.Procedure
+
+. Review an existing {product-title} release image of the single-architecture hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get hostedcluster/<hosted_cluster_name> -o jsonpath='{.spec.release.image}' <1>
+----
++
+<1> Replace `<hosted_cluster_name>` with your hosted cluster name.
++
+.Example output
+[source,terminal]
+----
+quay.io/openshift-release-dev/ocp-release:<4.y.z>-x86_64 <1>
+----
+<1> Replace `<4.y.z>` with the supported {product-title} version that you use.
+
+. In your {product-title} release image, if you use the digest instead of a tag, find the multi-architecture tag version of your release image:
+
+.. Set the `OCP_VERSION` variable for the {product-title} version by running the following command:
++
+[source,terminal]
+----
+$ OCP_VERSION=$(oc image info quay.io/openshift-release-dev/ocp-release@sha256:ac78ebf77f95ab8ff52847ecd22592b545415e1ff6c7ff7f66bf81f158ae4f5e -o jsonpath='{.config.config.Labels["io.openshift.release"]}')
+----
+
+.. Set the `MULTI_ARCH_TAG` variable for the multi-architecture tag version of your release image by running the following command:
++
+[source,terminal]
+----
+$ MULTI_ARCH_TAG=$(skopeo inspect docker://quay.io/openshift-release-dev/ocp-release@sha256:ac78ebf77f95ab8ff52847ecd22592b545415e1ff6c7ff7f66bf81f158ae4f5e | jq -r '.RepoTags' | sed 's/"//g' | sed 's/,//g' | grep -w "$OCP_VERSION-multi$" | xargs)
+----
+
+.. Set the `IMAGE` variable for the multi-architecture release image name by running the following command:
++
+[source,terminal]
+----
+$ IMAGE=quay.io/openshift-release-dev/ocp-release:$MULTI_ARCH_TAG
+----
+
+.. To see the list of multi-architecture image digests, run the following command:
```

Comment:
Should this step come before the previous step? Or is it a verification step? Maybe state "verification" if so.

Reply:
@dfitzmau +1 to Bryan.